src/libraries/System.Private.Xml/src/System/Xml/Serialization/XmlSerializer.cs

```diff
@@ -150,29 +150,38 @@ private static XmlSerializerNamespaces DefaultNamespaces
             }
         }
 
+        // Linker warning messages
+        internal const string LinkerSerializationMessage = "Members from serialized types may be trimmed if not referenced directly";
```

Comment:
Let's avoid the term "Linker" spreading outside the mono/linker repo.

The tool name is just plain unfortunate and we should keep it an implementation detail until we can fully get rid of it. It's already ambiguous in Xamarin (see e.g. here: https://github.com/xamarin/xamarin-macios/issues/10251#issuecomment-794368281 where it needs to disambiguate between "link native code" and "il link" instead of being able to use the standard computer term that everyone outside .NET understands for what it is - "link").

Reply:
removed.